test/test_variable_size_binary_layout.cpp

```diff
@@ -39,17 +39,18 @@ namespace sparrow
 
         static constexpr std::array<std::string_view, 4> words = {"you", "are", "not", "prepared"};
         array_data m_data;
-        // TODO: replace R = std::string_view with specific reference proxy
 
     private:
 
-        std::int64_t* offset()
-        {
-            SPARROW_ASSERT_FALSE(m_data.buffers.empty());
-            return m_data.buffers[0].data<std::int64_t>();
-        }
+       // TODO This isn't used at the moment, are we supposed to use it later or should we remove it to avoid polluting the file?
+//         std::int64_t* offset()
+//         {
+//             SPARROW_ASSERT_FALSE(m_data.buffers.empty());
+//             return m_data.buffers[0].data<std::int64_t>();
+//         }
```

Comment:
This isn't used at the moment, are we supposed to use it later or should we remove it to avoid polluting the file?